examples/checkpoint_example_traversal.cc

```diff
@@ -149,3 +151,5 @@ int main(int, char**) {
 
   return 0;
 }
+
+/// [Serialization with custom traverser]
```

Comment:
Please fix the end-of-file newline, so utilities don't complan

Reply:
Ok Fixed.